content_scripts/link_hints.js

```diff
@@ -1213,6 +1213,11 @@ const LocalHints = {
       possibleFalsePositive = true;
     }
 
+    if (!isClickable && className?.toLowerCase().includes("btn")) {
+      isClickable = true;
+      possibleFalsePositive = true;
+    }
```

Comment:
This could be part of the same if statement by doing something like:
`if (!isClickable && (className?.toLowerCase().includes("button") || className?.toLowerCase().includes("btn")))`

Reply:
Agreed, I've made a change to address this.